Fix Logging on Volume Creation

akutz opened the pull request:
This patch handles fixes #478 by centralizing logging of volume creation activity into the volume route source away from the driver implementation. The error state is also handled in the route source meaning drivers should no longer log the error directly in their implementation of `VolumeCreate`.

codenrhoden reviewed (approved):
Very much :+1: Nice improvement.

And you fixed the typo while I was reviewing it. :)

akutz committed:
Fix Logging on Volume Creation

This patch handles fixes #478 by centralizing logging of volume creation
activity into the volume route source away from the driver
implementation. The error state is also handled in the route source
meaning drivers should no longer log the error directly in their
implementation of VolumeCreate.
